projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.html

```diff
@@ -6,11 +6,18 @@
       [max]="maxQuantity"
       [control]="addToCartForm.get('quantity')"
     ></cx-item-counter>
-    <span class="info">{{
-      hasStock
-        ? ('addToCart.inStock' | cxTranslate)
-        : ('addToCart.outOfStock' | cxTranslate)
-    }}</span>
+
+    <span *ngIf="showInventory; else showLabelOnly" class="info"
+      >{{ getInventory() }} {{ 'addToCart.inStock' | cxTranslate }}</span
```

Comment:
I see that you return an empty string  for `getInventory`

can't you re-use the old template and append the getInventory function to it?

Reply:
This case is needed for an edge case. (Force In Stock status for a product with 0 availability/quantity ).